app/Imports/ImportCustomersCustomer.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Imports;
+
+use App\Models\Customers;
+use Maatwebsite\Excel\Concerns\ToModel;
+use Maatwebsite\Excel\Concerns\WithHeadingRow;
+
+class ImportCustomersCustomer implements ToModel, WithHeadingRow
+{
+    /**
+    * @param array $row
+    *
+    * @return \Illuminate\Database\Eloquent\Model|null
+    */
+    public function model(array $row)
+    {
+        return new Customers([
```

Comment:
![image](https://user-images.githubusercontent.com/32955000/182066486-6427006e-aee2-49b4-9186-c83b99c4e915.png)
Temporarily change catch error message to `$e->getMessage()` to see this error.

Reply:
This kind of error happens when user has no input in a row on import file but is not `nullable` in migrations.  Enhanced the validation. Will implement on other import/export if approved.